protos/sources/vega/commands/v1/commands.proto

```diff
@@ -113,12 +113,40 @@ message LiquidityProvisionSubmission {
   string reference = 6;
 }
 
+// A liquidity provision submitted for a given spot market
+message SpotLiquidityProvisionSubmission {
+  // Market ID for the order, required field.
+  string market_id = 1;
+  // Specified as a unitless number that represents the amount of quote asset of the market.
+  // This field is an unsigned integer scaled using the asset's decimal places.
+  string buy_commitment_amount = 2;
+  // Specified as a unitless number that represents the amount of base asset of the market.
+  // This field is an unsigned integer scaled using the asset's decimal places.
+  string sell_commitment_amount = 3;
+  // Nominated liquidity fee factor, which is an input to the calculation of taker fees on the market, as per setting fees and rewarding liquidity providers.
+  string fee = 4;
+  // Set of liquidity sell orders to meet the liquidity provision obligation.
+  repeated vega.LiquidityOrder sells = 5;
+  // Set of liquidity buy orders to meet the liquidity provision obligation.
+  repeated vega.LiquidityOrder buys = 6;
```

Comment:
No need for those?

Reply:
because of SLA? well, we're not in SLA land yet are we?

Reply:
well ye, but there's also no order to deploy right as we do not use the hold lp mecanism for spot? so they will be unused?